src/pages/guide/screens/FAQs/FAQs.tsx

```diff
@@ -1,15 +1,32 @@
+import { useRef } from 'react';
 import { useLiveChat } from '@/hooks/custom-hooks';
 import { Localize, useTranslations } from '@deriv-com/translations';
 import { Accordion, Text, useDevice } from '@deriv-com/ui';
 import { URLConstants } from '@deriv-com/utils';
+import './FAQs.scss';
 
-const FAQs = () => {
+type TFAQsProps = {
+    guideContentRef: React.RefObject<HTMLDivElement>;
+};
+
+const FAQs = ({ guideContentRef }: TFAQsProps) => {
     const { LiveChatWidget } = useLiveChat();
     const { isDesktop } = useDevice();
     const { localize } = useTranslations();
+    const accordionRefs = useRef<HTMLDivElement[]>([]);
+
+    const handleScrollToAccordion = (index: number) => {
+        const offsetByValue = isDesktop ? 0 : 100;
```

Comment:
how was this 100 calculated? 🤔

Reply:
mostly through trial and error 😛 but i mostly needed to make sure it didn't go underneath the header for the full page modal in mobile and tablet